requirements_dev.txt

```diff
@@ -6,9 +6,9 @@ flake8>=3.7.8
 tox>=3.14.0
 coverage>=4.5.4
 Sphinx>=2.2.0
-twine?=2.2.0
+twine>=2.2.0
 pytest>=5.1.3
 pytest-runner>=5.1
 pandas>=0.25.1
 pandas_flavor>=0.1.2
-humanize>=0.15.2
+humanize>=0.5.0
```

Comment:
@cmdavis4 can you explain the version change?

Reply:
0.15 doesn't seem to be an available version of humanize and it causes pip to break on this line - from the pypi page, available versions are 0.1, 0.2, 0.3, 0.4, 0.5, 0.5.1, and (as of two days ago) 1.0.0. I picked 0.5 somewhat arbitrarily, happy to change it, just wanted to make it something that works with pip.